app.yaml

```diff
@@ -36,3 +36,14 @@ automatic_scaling:
 
 env_variables:
   NODE_ENV: production
+
+# Temporary workaround for a Cloud SDK bug.
+# Ensures that node_modules directory is not uploaded (the other entries are the
+# default values for skip_files).
+skip_files:
```

Comment:
Done. `.git` comes for free, since the default skips Unix hidden files. Added *.log